samples/subsys/power/device_pm/sample.yaml

```diff
@@ -4,3 +4,15 @@ tests:
   sample.power.ospm.dev_idle_pm:
     platform_whitelist: nrf52840_pca10056 nrf52_pca10040
     tags: power
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "Device PM sample app start"
+        - "parent resuming\\.\\."
```

Comment:
The double backslashes suck, these should really be Python "raw" strings. Could they be? Not in this PR I guess but at some point in the future.

Reply:
It's YAML, actually.  The first backslash is required (we're matching a literal period character and not any character), the second is to get it through YAML which also interprets backslash escapes and needs a second one to pass it through to the next layer.

Surely YAML has a better string delimeter that does what we want too, but I don't know it and quite frankly would rather spend my day reading C code in ANSI trigraphs than try to look up YAML syntax. :)

Reply:
I just did lookup YAML syntax briefly. Single quotes should Just Work:
```
regex:
        - 'Device PM sample app start'
        - 'parent resuming\.\.'
```
According to this page http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html
> A double quoted scalar is the only scalar style where you can use escape (backslash) sequences.

So using double-quotes here seems like shooting yourself in the foot.